micromagneticdata/abstract_drive.py

```diff
@@ -103,7 +103,7 @@ def m0(self):
         Field(...)
 
         """
-        return df.Field.fromfile(self._m0_path)
+        return df.Field.fromfile(str(self._m0_path))
```

Comment:
Is this because `Field.fromfile` does not accept `Path` type?

Reply:
Yes. I would suggest to keep it for now and switch to `pathlib.Path` everywhere (in all packages) separately.

Reply:
+1